.ci/get_version_from_git.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
```

Comment:
I would probably move this script into the `hack` directory, instead, since we have that to hold various scripts.